src/parser/classes/livechat/items/LiveChatPaidSticker.ts

```diff
@@ -42,7 +43,8 @@ class LiveChatPaidSticker extends YTNode {
     this.author_name_text_color = data.authorNameTextColor;
     this.sticker = Thumbnail.fromResponse(data.sticker);
     this.purchase_amount = new Text(data.purchaseAmountText).toString();
-    this.context_menu = new NavigationEndpoint(data.contextMenuEndpoint);
+    this.menu_endpoint = new NavigationEndpoint(data.contextMenuEndpoint);
+    this.context_menu = this.menu_endpoint;
```

Comment:
Like others, this should be named menu_endpoint, not context_menu. To avoid breaking changes, I make both available.